htdocs/install/mysql/migration/20.0.0-21.0.0.sql

```diff
@@ -282,3 +282,5 @@ CREATE TABLE llx_product_customer_price_extrafields (
 ) ENGINE=innodb;
 
 ALTER TABLE llx_product_customer_price_extrafields ADD UNIQUE INDEX uk_product_customer_price_extrafields (fk_object);
+ALTER TABLE llx_facture ADD COLUMN payment_reference varchar(25) AFTER date_lim_reglement;
```

Comment:
An invoice can be paid by several payment. So data for payments should be store into the table llx_payment and table for invoice.
What is the use case of this field ?